src/metrics/metrics.md

```diff
@@ -2,10 +2,59 @@
 
 Here we provide definitions and information about our standard metrics.
```

Comment:
Should we link to the terminology section (https://docs.telemetry.mozilla.org/concepts/terminology.html) in this introduction? I think some of the material there may be helpful in interpreting what's in DAU.

Reply:
Sure, adding.  I think that section needs a lot more material - might be something we can work on as part of this initiative - but it's still useful to reference.